packages/app-layout/src/vaadin-app-layout.js

```diff
@@ -400,12 +409,21 @@ class AppLayout extends ElementMixin(ThemableMixin(mixinBehaviors([IronResizable
     this._updateOverlayMode();
 
     window.addEventListener('close-overlay-drawer', this.__closeOverlayDrawerListener);
+
+    this.$.drawer.addEventListener('keydown', this.__onDrawerKeyDown);
```

Comment:
Since `drawer` is a static element in the shadow root of this, there shouldn't be a need to toggle the "keydown" event listener on and off. The listener can be bound in the template with `on-keydown="__onDrawerKeyDown"` and  `this.__onDrawerKeyDown = this.__onDrawerKeyDown.bind(this);` can be removed

Reply:
Good point, fixed.